storage/replica_command.go

```diff
@@ -505,12 +505,11 @@ func (r *Replica) EndTransaction(
 	return reply, externalIntents, nil
 }
 
-// resolveExplicitIntents resolve any explicit intents and persists e
-// transaction record with updated status. All that are local to this
-// range get resolved synchronously in the same batch. The remainder
-// are collected and returned so that they can be handed off to
-// asynchronous processing.
-func (r *Replica) resolveExplicitIntents(batch engine.Engine, ms *engine.MVCCStats, args roachpb.EndTransactionRequest, txn *roachpb.Transaction) ([]roachpb.Intent, error) {
+// resolveExplicitIntents resolve any explicit intents. All that are
```

Comment:
I still don't know what the explanation for the word "explicit" in this context is.